read.go

```diff
@@ -226,24 +228,24 @@ func readNativeFrames(d dicomio.Reader, parsedData *Dataset, fc chan<- *frame.Fr
 		buf := make([]int, int(pixelsPerFrame)*samplesPerPixel)
 		for pixel := 0; pixel < int(pixelsPerFrame); pixel++ {
 			for value := 0; value < samplesPerPixel; value++ {
+
+				// read into pre-allocated pixel buffer
+				n, err := d.Read(pixelBuf)
+				if err != nil {
+					return nil, bytesRead,
+						fmt.Errorf("could not read uint%d from input: %w", bitsAllocated, err)
+				}
+				if n < bitsAllocated/8 {
```

Comment:
since this is used in L216 as well, can we make this a variable with a name (that might enhance readability) like  `intBytes` or `uintBytes`?

Reply:
how about `bytesAllocated`?